sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/KeyRotationPolicy.java

```diff
@@ -42,54 +37,100 @@ public String getId() {
     }
 
     /**
-     * Get the {@link KeyRotationPolicy policy's} created time in UTC.
+     * Get the actions that will be performed by Key Vault over the lifetime of a key.
      *
-     * <p>May be undefined if a {@link KeyRotationPolicy policy} has not been explicitly set.</p>
+     * <p>You may also pass an empty array to restore to its default values.</p>
      *
-     * @return The {@link KeyRotationPolicy policy's} created time in UTC.
+     * @return The {@link KeyRotationLifetimeAction actions} in this {@link KeyRotationPolicy policy}.
      */
-    public OffsetDateTime getCreatedOn() {
-        return this.createdOn;
+    public List<KeyRotationLifetimeAction> getLifetimeActions() {
+        return this.keyRotationLifetimeActions;
     }
 
     /**
-     * Get the {@link KeyRotationPolicy policy's} last updated time in UTC.
+     * Set the actions that will be performed by Key Vault over the lifetime of a key.
      *
-     * <p>May be undefined if a {@link KeyRotationPolicy policy} has not been explicitly set.</p>
+     * <p>You may also pass an empty array to restore to its default values.</p>
      *
-     * @return The {@link KeyRotationPolicy policy's} last updated time in UTC.
+     * @param keyRotationLifetimeActions The {@link KeyRotationLifetimeAction actions} to set.
+     *
+     * @return The updated {@link KeyRotationPolicy} object.
      */
-    public OffsetDateTime getUpdatedOn() {
-        return this.updatedOn;
+    public KeyRotationPolicy setLifetimeActions(List<KeyRotationLifetimeAction> keyRotationLifetimeActions) {
+        this.keyRotationLifetimeActions = keyRotationLifetimeActions;
+
+        return this;
+    }
+
+    /**
+     * Get the optional key expiration period used to define the duration after which a newly rotated key will expire.
+     * It should be defined as an ISO 8601 duration. For example, 90 days would be formatted as follows: "P90D", 3
+     * months would be "P3M", 48 hours would be "PT48H" and 1 year and 10 days would be "P1Y10D".
+     *
+     * @return The expiration time in ISO 8601 format.
```

Comment:
ISO 8601 *duration* format. It's a separate section. The majority of ISO 8601 covers timestamps not durations.